frontend/src/components/courses/search/ProtectedSearchStepper.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import Stepper from 'primevue/stepper';
+import StepperPanel from 'primevue/stepperpanel';
+import Button from 'primevue/button';
+import { useI18n } from 'vue-i18n';
+
+/* Composable injections */
+const { t } = useI18n();
+</script>
+
+<template>
+    <Stepper orientation="vertical">
+        <StepperPanel :header="t('protectedCourses.screen1.title')">
+            <template #content="{ nextCallback }">
+                <div class="flex flex-column h-12rem">
+                    <div
+                        class="border-4 surface-border border-round flex-auto flex justify-content-center align-items-center"
+                    >
+                        {{ t('protectedCourses.screen1.content') }}
+                    </div>
+                </div>
+                <div class="flex py-4 justify-content-end">
+                    <Button
+                        icon="pi pi-arrow-right"
+                        iconPos="right"
+                        @click="nextCallback"
+                        text
+                        class="p-button-rounded"
+                    />
+                </div>
+            </template>
+        </StepperPanel>
+        <StepperPanel :header="t('protectedCourses.screen2.title')">
+            <template #content="{ prevCallback, nextCallback }">
+                <div class="flex flex-column h-12rem">
+                    <div
+                        class="border-4 surface-border border-round flex-auto flex justify-content-center align-items-center"
+                    >
+                        {{ t('protectedCourses.screen2.content') }}
+                    </div>
+                </div>
+                <div class="flex py-4 gap-2">
```

Comment:
Arrow to next StepperPanel is not all the way to the left here, while it is in the first StepperPanel. Maybe this can be adjusted for aesthetic purposes :).